nexus/lib/nexus/peer.ex

```diff
@@ -84,14 +83,24 @@ defmodule Nexus.Peer do
     GenServer.call(registry_id(id), {:add_ice_candidate, body})
   end
 
-  @spec send_pli(id()) :: :ok
-  def send_pli(id) do
-    GenServer.cast(registry_id(id), :send_pli)
+  @spec peer_added(id()) :: :ok
+  def peer_added(id) do
+    PubSub.broadcast(Nexus.PubSub, "room", {:peer_added, id})
+  end
+
+  @spec peer_removed(id()) :: :ok
+  def peer_removed(id) do
+    PubSub.broadcast(Nexus.PubSub, "room", {:peer_removed, id})
   end
```

Comment:
This shouldn't be a part of Peer API. Let's move this to the Room module as it's the only one who calls those funcitons